bench/workgen/runner/example_prepare_stress.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2020 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+# This benchmark is designed to stress disk access to the history store file using timestamps.
+# This is achieved through:
+#   - Long running transactions consisting of read and update operations.
+#   - Low cache size (~20%) for a reasonably sized WT table with large documents.
+#   - Pareto distribution for operations in long running transactions. This will cause
+#     skewed access for a selective set of keys in WT table.
+#   - Relatively large number of read operation threads to stress cache.
+#   - It uses prepare transactions.
+
+###################################################################################################
+'''
+# wtperf options file: evict btree configuration
```

Comment:
If this file was derived from evict-btree-hs.py, I would say that somewhere in a comment, and state generally how it was changed.  I think whenever you make a copy of a (file/function/project) that is more than mostly unchanged, it's good to do this - it can allow a future developer to diff the old and new, which can be very helpful. If a bug or flaw is corrected in the "original", it is more likely to be corrected in the derived as well.

Reply:
Added a comment section stating the origin of the file and the list of changes done to this script.